.drone.nightly.yml

```diff
@@ -4,37 +4,37 @@ clone:
 
 build:
   integration-test:
-    image: $${TEST_BUILD_IMAGE=gcr.io/eminent-nation-87317/vic-integration-test:1.30}
+    image: ${TEST_BUILD_IMAGE=gcr.io/eminent-nation-87317/vic-integration-test:1.30}
```

Comment:
I dont think we are updating the nightly builds to 05 yet. @mhagen-vmware can provide details